packages/twenty-ui/src/input/button/components/Button.tsx

```diff
@@ -408,18 +408,21 @@ export const Button = ({
   soon = false,
   disabled = false,
   justify = 'flex-start',
-  focus = false,
+  focus: propFocus = false,
   onClick,
   to,
   target,
   dataTestId,
   hotkeys,
   ariaLabel,
+  type,
 }: ButtonProps) => {
   const theme = useTheme();
 
   const isMobile = useIsMobile();
 
+  const [isFocused, setIsFocused] = React.useState(propFocus);
```

Comment:
I don't see the benefit of this state. Could you please explain it to me? Ideally, the focus should not be set through a prop, as the prop will never be the source of truth. Nonetheless, the `isFocused` state doesn't seem to do anything here. Did I miss something?

Reply:
I 100% agree. Historically focus was a props. I didn't to break the current implementation. But I needed to set the right state when you're focused on it. That's why I did this hack